hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -200,7 +200,7 @@ protected void initializeIncomingRecordsMap() {
       LOG.info("MaxMemoryPerPartitionMerge => " + memoryForMerge);
       this.keyToNewRecords = new ExternalSpillableMap<>(memoryForMerge, config.getSpillableMapBasePath(),
           new DefaultSizeEstimator(), new HoodieRecordSizeEstimator(tableSchema),
-          config.getSpillableDiskMapType());
+          config.getSpillableDiskMapType(), config.isBitCaskDiskMapCompressionEnabled());
```

Comment:
Do you wanna make change in HoodieMergedLogRecordScanner as well ? Or thats planned for a follow up PR

Reply:
Good point, will be done in a follow up PR https://issues.apache.org/jira/browse/HUDI-2044